### ISSUE TITLE
why doesn't Mac client prompt for password?

### ISSUE THREAD

**Issue by robnagler** [keybase/client#5230]
I can't figure out how to get Keybase to prompt for a password when I start the app. It just starts with kbfs mounted without any prompt for my passphrase.

**Comment by cjb** [comment]
It's using a secret derived from your passphrase that's stored in your macOS Keychain (Keychain Access control panel).

**Comment by robnagler** [comment]
I don't want that. I don't store anything in the Keychain if I can help it. Is there a way to disable this?

**Comment by cjb** [comment]
No, I don't believe so (unless you can disallow Keybase from storing in the keychain at the OS level?). CC'ing @malgorithms to note the feature request.

**Comment by maxtaco** [comment]
BTW, we don't store to iCloud keychain, on the local OSX keychain.

**Comment by robnagler** [comment]
I modified keybase's access to the login keychain value "keybase" by removing keybase's access to "keybase" and requiring a password prompt:

![keybase-access](https://cloud.githubusercontent.com/assets/5495179/21575077/4542fc06-cebe-11e6-86d6-7b3141db729f.jpg)

However, now I get prompted 4 times when I want to login to keybase. If I do Deny access (4 times), it will prompt me for my keybase password.

If I remove the "keybase" entry in the login keychain, it gets re-created the next time I run the app. This might be a place where the Keybase app could use a preference, or at least, if it sees the "keybase" entry in the login keychain deleted, it shouldn't re-create it.

**Comment by evert** [comment]
+1 here, I always configured my gpg pre-keybase to always prompt for a password. I only use it for things that need to actually be done really securely and having it tied to my operating system password drastically reduces the security for me.

If it prompted with passwords by default, I would for example feel good enough about the keybase chat to send passwords and other secrets, but now I need to trust others to remove old messages and always lock their machine when they left it.
